### DISCUSSION TITLE
Prisma’s jsonProtocol compatibility

### DISCUSSION THREAD

**Discussion by JClackett** [MichalLytek/typegraphql-prisma#377] [General]
Hey just a quick one, will the upcoming changes to prisma's internals regarding their jsonProtocol affect this package in anyway?

**Comment by MichalLytek** [comment]
Have you tried using it before asking a question? 😅

**Reply by JClackett** [reply]
Yes I did, but I was wondering if they'll be any future issues as I saw something about big internal changes and wanted to check that this package will always work with prisma

**Reply by MichalLytek** [reply]
Internal Prisma communication stuff should not interfere with generating code based on Prisma schema.

**Comment by MichalLytek** [comment]
Closing as checked and it has no impact on the `typegraphql-prisma` luckily 💪
